enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,164 @@
+---
+title: install-ca-management
+authors:
+  - "@bparees"
+reviewers:
+  - "@sdodson"
+  - "@abhinavdahiya"
+  - "@adambkaplan"
+  - "@dmage"
+approvers:
+  - "@sdodson"
+  - "@crawford"
+creation-date: 2020-08-03
+last-updated: 2020-08-03
+status: provisional
+---
+
+# Install CA Management
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Customers have a need to provide additional CAs to their cluster.  Ideally they can do this at 
+install time so that everything works by the time the cluster comes up (if they do not, the 
+cluster may install but things like imagestreams may not import, builds may not work, or other 
+components may not function as expected).
+
+Today the installer collects an “additionalTrustBundle”, and optionally proxy host names.
+
+The additionalTrustBundle is put(not sure where exactly?) and consumed directly by some set of 
```

Comment:
> put(not sure where exactly?)

https://github.com/openshift/installer/blob/fcf8f4517c79a65ca4f14f1aa5af48f66cea3df9/pkg/asset/manifests/additionaltrustbundleconfig.go#L65-L75

`openshift-config/user-ca-bundle` ConfigMap

Reply:
thanks, will update w/ this detail